index.js

```diff
@@ -6,6 +6,14 @@ app.get('/', (req, res) => {
   res.send('Hello World!')
 })
 
+app.get('/ping', (req, res) => {
+  res.send('pon g')
```

Comment:
[GPT-REVIEW] There seems to be a typo in the response string 'pon g'. It should be 'pong' to correctly respond to a 'ping' request.